tools/clang/test/CodeGenHLSL/allocateRayQuery2.hlsl

```diff
@@ -0,0 +1,15 @@
+// RUN: %dxc -T lib_6_9 %s | FileCheck %s 
```

Comment:
I think we just need another RUN line with `-fcgl` and additional checks to verify the initial RayQuery constructor translation in `CGHLSLMSFinishCodeGen.cpp`.